python/graphstorm/dataloading/dataloading.py

```diff
@@ -357,7 +374,9 @@ def target_eidx(self):
 class GSgnnLinkPredictionDataLoader(GSgnnLinkPredictionDataLoaderBase):
     """ Link prediction minibatch dataloader
 
-    The negative edges are sampled uniformly.
+    GSgnnLinkPredictionDataLoader samples GraphStorm edge dataset into an iterable over mini-batches 
+    of samples. In each batch, pos_graph and neg_graph are sampled subgraph for positive and 
+    negative edges, respectively. Given positive edges, the negative edges are sampled uniformly. 
```

Comment:
Could you explain a little about the uniform sampling? Maybe refer to the DGL's negative sampling document or add a link pointing to it?